function/.eslintrc.json

```diff
@@ -1,5 +1,12 @@
 {"extends": "strongloop",
  "parserOptions": {
-   "ecmaVersion": 6
+   "ecmaVersion": 2017
+ },
+ "env": {
+  "es6": true
+},
```

Comment:
necessary, otherwise lint complains about the bare use of `Promise`, which is an es6 global.